src/agent/onefuzz-telemetry/src/lib.rs

```diff
@@ -492,23 +492,22 @@ pub fn set_property(entry: EventData) {
     }
 }
 
-fn local_log_event(event: &Event, properties: &[EventData]) {
-    let as_values = properties
+pub fn format_events(events: &[EventData]) -> String {
+    events
         .iter()
         .map(|x| x.as_values())
         .map(|(x, y)| format!("{}:{}", x, y))
         .collect::<Vec<String>>()
-        .join(" ");
-    log::log!(log::Level::Info, "{} {}", event.as_str(), as_values);
```

Comment:
So this line was the core problem, right? `log::log!()` got expanded in `onefuzz-telemetry`, and not in call sites in the consumer crate. As a result, it'd pick up the wrong module name.